application/src/test/java/org/opentripplanner/routing/algorithm/mapping/CarPickupSnapshotTest.java

```diff
@@ -0,0 +1,71 @@
+package org.opentripplanner.routing.algorithm.mapping;
+
+import au.com.origin.snapshots.junit5.SnapshotExtension;
```

Comment:
Btw how is this dependency imported into our project?

Reply:
It's an existing dependency: 
https://github.com/opentripplanner/OpenTripPlanner/blob/5b5d92fe7e81c2fab07f2ff6ae8f7488427be71c/pom.xml#L466-L470